src/ghga_datasteward_kit/file_deletion.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Provides functionality to request file service data deleteion."""
+
+import logging
+from pathlib import Path
+
+import httpx
+from pydantic import Field
+from pydantic_settings import BaseSettings
+
+from ghga_datasteward_kit.utils import DELETION_TOKEN, load_config_yaml
+
+log = logging.getLogger(__name__)
+
+
+class FileDeletionConfig(BaseSettings):
+    """Config for calling the PCS file deletion endpoint"""
+
+    file_deletion_baseurl: str = Field(
+        default=...,
+        description=(
+            "Base URL under which the delete /files endpoint is available."
+            + " This is an endpoint exposed by GHGA Central. This value is provided by"
+            + " GHGA Central on demand."
+        ),
+    )
+    file_deletion_endpoint: str = Field(
+        default="/files",
+        description=(
+            "Path to the PCS endpoint (relative to baseurl) expecting a delete request including"
+            + " the ID of the file for which data should be deleted in the file services."
+        ),
+    )
+
+
+def main(*, file_id: str, config_path: Path):
+    """Call PCS to delete all data in the file services for the given file ID."""
+    config = load_config_yaml(path=config_path, config_cls=FileDeletionConfig)
+
+    url = f"{config.file_deletion_baseurl}{config.file_deletion_endpoint}/{file_id}"
```

Comment:
Better use something like urljoin. I think we changed it already elsewhere because we cannot rely on the config having "end slashes" or not.

Reply:
There's at least one other point in the codebase where it's done this way.
I'll try to fix all of them

Reply:
urljoin does not like construction from three parts, i.e. `config.file_deletion_endpoint` still behaves differently depending on whether it has a trailing slash or not.

I went with 
```
    base = config.file_deletion_baseurl.rstrip("/")
    endpoint = config.file_deletion_endpoint.strip("/")
    url = f"{base}/{endpoint}/{file_id}"
```
instead